src/wasm/wasm-emscripten.cpp

```diff
@@ -326,7 +351,7 @@ void EmscriptenGlueGenerator::generatePostInstantiateFunction() {
     POST_INSTANTIATE, std::vector<NameType>{}, Type::none, {});
   wasm.addFunction(post_instantiate);
 
-  if (Function* F = generateAssignGOTEntriesFunction()) {
+  if (Function* F = generateAssignGOTEntriesFunction(true /*isSideModule*/)) {
```

Comment:
The chromium/llvm style for this is `/*isSideModule=*/true` (with no spaces I think).   Not sure if this is a common pattern in binaryen?

Reply:
Looking around, it seems like we mostly use bools with `.setX()` functions. I'll adapt to that.